mindone/utils/load_params.py

```diff
@@ -0,0 +1,136 @@
+import copy
+import os
+from typing import List, Optional, Union
+
+import mindspore as ms
+import mindspore.nn as nn
+from mindspore import Parameter
+from mindspore import log as logger
+
+# from mindspore._checkparam import Validator
+from mindspore.train.serialization import _load_dismatch_prefix_params, _update_param
+
+
+def load_param_into_net_with_filter(
+    net: nn.Cell, parameter_dict: dict, strict_load: bool = False, filter: Optional[List] = None
+):
```

Comment:
Not the same. It's to filter the parameters in the network rather than checkpoints for loading. For example, loading unet 2d checkpoint to UNet3D cell, hundreds of warnings will emerge if use `ms. load_param_into_net ()` which makes it hard to spot the real bug